hack/verify-lint.sh

```diff
@@ -30,5 +30,5 @@ for d in $(find . -type d -a \( -iwholename './pkg*' -o -iwholename './cmd*' \))
 		 --cyclo-over=60 \
 		 --dupl-threshold=100 \
 		 --tests \
-		 --deadline=300s "${d}"
+		 --deadline=600s "${d}"
```

Comment:
I updated https://github.com/kubernetes-incubator/cri-containerd/pull/77. Let's see whether it's a travis error, maybe it's fixed today.